src/modules/75-ce/RouteDestinations.tsx

```diff
@@ -755,6 +756,10 @@ const CERoutes: React.FC = () => {
                 OverviewAddCluster,
                 RecommendationFilters
               }}
+              customAPIHooks={{
+                useGetAggregatedUsers: useGetAggregatedUsers,
+                useGetConnector: useGetConnector
```

Comment:
TOTALLY OPTIONAL... you could simplify this a wee bit
```suggestion
                useGetAggregatedUsers,
                useGetConnector
```